client/client-multi/client-c/toolchains/edison.cmake

```diff
@@ -0,0 +1,85 @@
+#
+# Copyright 2014-2016 CyberVision, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+if(NOT DEFINED SDK_32bits)
+	set(SDK_32bits FALSE)
+endif()
+
+#Set the target parameters
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_VERSION 3.10.17-poky-edison+)
+set(CMAKE_SYSTEM_PROCESSOR i686)
+
+if(NOT EXISTS "$ENV{EDISON_SDK_ROOT}")
+    if(WIN32)
+        if(${SDK_32bits})
+            set(EDISON_SDK_ROOT C:/Edison/poky-edison-eglibc-i686-edison-image-core2-32-toolchain-1.7.2)
+        else()
+            set(EDISON_SDK_ROOT C:/Edison/poky-edison-eglibc-x86_64-edison-image-core2-32-toolchain-1.7.2)
+        endif()
+    else(WIN32)
+        set(EDISON_SDK_ROOT /opt/poky-edison/1.6)
+    endif(WIN32)
+else(NOT EXISTS "$ENV{EDISON_SDK_ROOT}")
+    set(EDISON_SDK_ROOT $ENV{EDISON_SDK_ROOT})
+endif(NOT EXISTS "$ENV{EDISON_SDK_ROOT}")
+
+message(STATUS "EDISON_SDK_ROOT: ${EDISON_SDK_ROOT}")
+
+#Set the host parameters
+if(WIN32)
```

Comment:
Then remove all the code that relates to Windows support. Let's avoid committing the code that does not work.